components/next/nextController.js

```diff
@@ -7,6 +7,14 @@ import interactions from '../interactions'
 const file = 'Next | Controller'
 
 const handleUser = async data => {
+  if (data.status === 'archived') {
+    try {
+      const success = await users.remove(data)
```

Comment:
```suggestion
      const success = await users.removeUser(data)
```

Reply:
Na verdade mesmo, eu estava vendo aqui, nem precisa do await, vc pode dar um retorn users.removeUser(data) direto, o arquivo que consome essa informação na verdade já utiliza o await, porém nem está assimilando uma variável para a resposta dessa função, mas só mesmo por garantia pode deixar direto:
```suggestion
      return users.remove(data)
```